pkg/proxy/iptables/proxier.go

```diff
@@ -300,15 +315,26 @@ func (scm *serviceChangeMap) update(namespacedName *types.NamespacedName, previo
 	return len(scm.items) > 0
 }
 
+// ipPort returns a string of the form "<ip>:<port>" (for IPv4)
+// or "[<ip>]:<port>" (for IPv6) for a given IP address and port
+func ipPort(ipStr string, port int) string {
+	ip := net.ParseIP(ipStr)
```

Comment:
net.JoinHostPort() seems to do exactly what this does, if we just fmt.Sprintf("%d", port) or stconv.Itoa(port)